internal/persistentdata/persistentdata.go

```diff
@@ -26,58 +34,58 @@ func New(fs afero.Afero, fn string, gen func() string) *Client {
 }
 
 // Get the data (generate if it doesn't exist) and return it
-func (c *Client) Get() (value string, newlyCreated bool, err error) {
-	if home, err := os.UserHomeDir(); err == nil {
-		// special case for having a persistent ID, read-only access
-		id, err := c.read(filepath.Join(home, ".cq"))
-		if err == ErrIsDirectory {
-			// finish early if directory encountered
-			return "", false, err
+func (c *Client) Get() (Value, error) {
+	var err error
```

Comment:
Why not define named return vars (`Get() (v Value, err error)`)? It doesn't have to be/become complicated if you don't do naked returns or just use them sparingly

Reply:
Added named return values. Good point about not using naked returns, never liked named return values because of them.